draft-regext-rdap-extensions.md

```diff
@@ -270,6 +387,18 @@ to technologies used by RDAP such as extended vCard/jCard properties.
 Such extensions exist to "mark" these usages and are often called "marker"
 extensions.
 
+For example, an extension may be used to signal desired processing of
+a `rel` attribute in a "links" array, where the `rel` value is registered in
+the Link Relations Registry (<https://www.iana.org/assignments/link-relations/link-relations.xhtml>).
+
+    "links": [
+      {
+        "value": "https://example.com/foo_bar",
+        "href" : "https://example.com/foo_baz",
+        "rel": "sideways"
+      }
+    ]
```

Comment:
Not clear from this example where the marker extension is leveraged. Is it "foo_baz" in "href"?